src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -163,6 +150,110 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     state?.deposit.cryptoAmount,
   ])
 
+  const getPreDepositInput: () => Promise<SendInput | undefined> = useCallback(async () => {
+    if (!(accountId && assetId && state?.deposit?.estimatedGasCrypto)) return
+
+    try {
+      const estimatedFees = await estimateFees(await getEstimateFeesArgs())
+      const amountCryptoBaseUnit = bnOrZero(state?.deposit.cryptoAmount).times(
+        bn(10).pow(asset.precision),
+      )
+      const quote = await getThorchainSaversQuote(asset, amountCryptoBaseUnit)
+
+      const sendInput: SendInput = {
+        cryptoAmount: '',
+        asset,
+        from: '', // Let coinselect do its magic here
+        to: maybeAccountAddress,
+        sendMax: true,
```

Comment:
After losing some hair trying to debug this and build a reconciliation Tx that works with @kaladinlight , decided to go with `sendMax` to populate that address with all other UTXOs.

Technically worse on the privacy factor, though the reconciliation by itself is already going to rug privacy, this just rugs it more.

Also slightly more gas, but realistically this isn't going to make a noticeable difference, and will have the added benefits of having that address already populated for future deposits/withdraws, unless the user does sends in the meantime, in which case we'll reconciliate again